library/src/main/java/com/novoda/downloadmanager/DownloadError.java

```diff
@@ -33,35 +45,37 @@ public String message() {
         return message;
     }
 
+    @Nullable
+    public Integer violatedRuleCode() {
```

Comment:
At the end of the chain, we'd retrieve the info (about a rule violation) from here